### ISSUE TITLE
Method description not shown when methods use arrows

### ISSUE THREAD

**Issue by xixixao** [typedoc2md/typedoc-plugin-markdown#670]
### What package is the bug related to?

typedoc-plugin-markdown

### Describe the issue

Works:

```ts
export type SomeType = {
  /**
   * METHOD DESCRIPTION
   *
   * @returns RETURN DESCRIPTION
   */
  signIn(): string;
};
```

Doesn't work:

```ts
export type SomeType = {
  /**
   * METHOD DESCRIPTION
   *
   * @returns RETURN DESCRIPTION
   */
  signIn: () => string;
};
```

In my case I don't need the arrow syntax, so I have a workaround, but it seems like a bug.

I verified this is a bug in this plugin and not in typedoc.

### TypeDoc configuration

Latest everything.

```sh
npx typedoc --plugin typedoc-plugin-markdown --entryPoints foo.ts --out ./docs
```

**Comment by xixixao** [comment]
Also btw it might be nice to give ppl some online playground (codepen or something) for easily sharing repros of issues.

**Comment by xixixao** [comment]
Actually I can't use the workaround, because that breaks type lints that check whether a method is bound or not.

**Comment by tgreyuk** [comment]
Fix in typedoc-plugin-markdown@4.2.5 .

**Comment by Artemeey** [comment]
> Fix in typedoc-plugin-markdown@4.2.5 .
> 
> > Also btw it might be nice to give ppl some online playground (codepen or something) for easily sharing repros of issues.
> 
> Thanks - yes that is not a bad idea.

Thanks!

However method description in index files still not output

**Comment by tgreyuk** [comment]
@Artemeey thanks - this is now fixed in  typedoc-plugin-markdown@4.2.6

**Comment by Artemeey** [comment]
> @Artemeey thanks - this is now fixed in typedoc-plugin-markdown@4.2.6

It's work!

But not all of them, example:

```
/**
 * Info
 */
export const exampleWork = (): void => { }; // exists description in index

/**
 * Info
 * @return info
 */
export const exampleNoWork = (): void => { }; // no exists description in index

```

**Comment by tgreyuk** [comment]
> But not all of them, example:

Ah - you are right - thanks. Will be fixed in next release.

**Comment by tgreyuk** [comment]
@Artemeey All issues should be resolved in Please try typedoc-plugin-markdown@4.2.7

**Comment by Artemeey** [comment]
> @Artemeey All issues should be resolved in Please try typedoc-plugin-markdown@4.2.7

It works, thanks!
